docs/content/latest/benchmark/performance-troubleshooting.md

```diff
@@ -108,10 +105,26 @@ W0325 06:47:13.033341 116514816 inbound_call.cc:208] Trace:
 0325 06:47:13.032168 (+    62us) inbound_call.cc:125] Queueing success response
 ```
 
+## Viewing real-time logs
+
+You can view logs of Yugabyte processes at a particular node (e.g. 127.0.0.1) at various ports:
+
+| Description | Address
+-------------|-----------|
+Master Metrics | 127.0.0.1:7000
```

Comment:
note: the metrics themselves will still be at `127.0.0.1:7000/metrics`, it's just that the port changes depending on which you want to look at...so really the split should be
- for these components, here are their webserver ports
- for each of them, you can view JSON vs prometheus metrics, on these endpoints